tmt/steps/discover/fmf.py

```diff
@@ -320,6 +323,44 @@ class DiscoverFmf(tmt.steps.discover.DiscoverPlugin[DiscoverFmfStepData]):
             how: fmf
             dist-git-source: true
 
+    Name Filter
+    ^^^^^^^^^^^
+
+    Use the ``test`` key to limit which tests should be executed by
+    providing regular expression matching the test name:
+
+    .. code-block:: yaml
+
+        discover:
+            how: fmf
+            test: ^/tests/area/.*
```

Comment:
Could we mention in a note what is used for matching? I assume it is `re.search`?

Reply:
Such info should be mentioned in the key help text, together with a reference to 

```
:ref:`regular-expressions`
```

It's how regular expressions should be documented - a key foo accepts regular expressions, uses "match" or "search" mode, see details on this page.